packages/interactivity/src/vdom.js

```diff
@@ -21,7 +21,7 @@ const directiveParser = new RegExp(
 		// (Optional) Match '--' followed by any alphanumeric charachters. It
 		// excludes underscore intentionally to prevent confusion, but it can
 		// contain multiple hyphens. E.g., "--custom-prefix--with-more-info".
-		'(?:--([a-z0-9][a-z0-9-]+))?$',
+		'(?:--([a-z0-9_-]+))?',
```

Comment:
Why did you remove the final `$`?

Reply:
Copy-pasted error. Is not affecting tests, so better leave it as it was.

Reply:
Great, thank you!